workshops/book-dash/meeting-notes/2023-09-05-WG.md

```diff
@@ -0,0 +1,57 @@
+# Book Dash [Provisional] Working Group Meeting 2023-09-05
+
+## Attendees 
+* Arielle Bennett
+* Emma Karoune 
+* (Apologies) Esther Plomp
+* (Apologies) Anne Lee Steele
+* (Apologies) Alex Araujo Alvarez
+
+## Agenda 
+* Working Group vs Planning Committee - governance and set up 
+* Applications for Nov 2023 Planning Committee
+* Activities for next Book Dash - discussion
+* Book Dash Hubs Set Up and Guidance
+
+## Decisions & Actions
+* Set up Working Group & separate Planning Committee in Nov following subsequent Book Dash - recommendation to core delivery team
+* Invite current and previous Planning Committee members to apply to WG in Nov - recommendation to core delivery team 
+* Accept all applications submitted to be part of this Planning Committee - recommendation to core delivery team
+* (Arielle) Upload notes to GitHub repo
+* (Arielle) communicate recommendations to core delivery team 
+
+## Notes
+* Book Dash working group vs planning committee team
+    * Assuming that we would still have RPM and RCM on planning committee, do need insight into operations and what is possible 
+    * (Recommendation) Planning committee - invite previous attendees to apply for next round as part of the follow up to next time 
+    * Working group - how are people selected/or how to do they apply? 
+        * Recommendations for basic criteria
+            * Have attended a Book Dash 
+            * [desirable] Have previously been on the planning committee at least once 
```

Comment:
```suggestion
            * Have previously been on the planning committee at least once 
```
I think for the working group it would be good to have people who have been on the planning committee before. For the planning committee I think it would be good to then just keep it at 'has attended the book Dash at least once'